packages/eslint-plugin-pf-codemods/lib/rules/v5/cardHeader-update-api.js

```diff
@@ -8,142 +8,146 @@ module.exports = {
     const removedCardImports = imports.filter((specifier) =>
       ["CardHeaderMain", "CardActions"].includes(specifier.imported.name)
     );
-    const cardHeaderImport = imports.find((specifier) => specifier.imported.name === "CardHeader");
+    const cardHeaderImport = imports.find(
+      (specifier) => specifier.imported.name === "CardHeader"
+    );
+
+    const findParents = (node, elementNameToFind) => {
+      const parents = {
+        current: node?.parent,
```

Comment:
current makes sense for the loop internal to this function but i'm not sure it makes sense as the return type - maybe more important if we did make this a helper func

also, should it be findParent() or findClosestParent() maybe as we are only going to return as soon as we find what we are looking for and not have an array of parents... or i suppose plural is because you are also finding the expression as a parent.. 

i suppose i'm showing my thoughts as i'm thinking and reading through this. it makes sense to me now, might make sense to just be overly verbose `getExpressionAndParentCardHeader` as that is all we are doing with this function and i think i'd go with `parent` instead of `current`.

but really all these words are just for a nit at this point 🤣

Reply:
Makes sense, I updated the naming of things for this. Let me know what you think